compatibility/bazel_tools/testing.bzl

```diff
@@ -6,277 +6,174 @@ load(
     "client_server_test",
 )
 load("@os_info//:os_info.bzl", "is_windows")
-load("//bazel_tools:versions.bzl", "version_to_name")
+load("//bazel_tools:versions.bzl", "cmp_version", "version_to_name")
 load("//:versions.bzl", "latest_stable_version")
 
 # Indexed first by test tool version and then by sandbox version.
 # Note that at this point the granularity for disabling tests
 # is sadly quite coarse. See
 # https://discuss.daml.com/t/can-i-disable-individual-tests-in-the-ledger-api-test-tool/226
 # for details.
```

Comment:
This comment seems outdated.

Reply:
How so? We still have the issue that the granularity is too coarse.

Reply:
> Indexed first by test tool version and then by sandbox version

IIUC they are no longer indexed by sandbox version.

Reply:
Ah I see, I’ll update it.

Reply:
updated

Reply:
Thank you!